pkg/config/loader.go

```diff
@@ -256,7 +256,7 @@ func init() {
 	r.Wallet.Store = "walletDB"
 	r.Consensus.DefaultLockTime = 1000
 	r.Consensus.DefaultAmount = 10
-	r.Consensus.ConsensusTimeOut = 5
+	r.Consensus.ConsensusTimeOut = DefaultConsensusTimeOutSeconds
```

Comment:
Unrelated to this PR, but as per Go's recommended naming schemes, `ConsensusTimeout` should just be `Timeout` in this context, as it already belongs to `Consensus` category.

Reply:
I agree, but this PR aimed to be backward compatible with the previous configuration files